### DISCUSSION TITLE
How to change the weather?

### DISCUSSION THREAD

**Discussion by dajiba6** [guardstrikelab/carla_apollo_bridge#55] [Q&A]
seems the weather is changed in bridge.py, but i dont know where is the weather_parameters.

**Comment by XiaoFei9704** [comment]
world = client.get_world()
weather = WeatherParameters(cloudiness=0.0, precipitation=0.0, precipitation_deposits=0.0, ...)
world.set_weather(weather)

Refers to:
https://carla.readthedocs.io/en/latest/python_api/#carlaworld
